### DISCUSSION TITLE
markdown option drops "%"

### DISCUSSION THREAD

**Discussion by K4zuki** [ickc/pantable#41] [Q&A] [answered]
### Condition

- `pantable` == 0.12.2
- python == 3.5

### Issue

following syntax drops single `%` character in cell

`````
```table
---
markdown: True
---
This,is,a table,%
%,%,%,&#37;
```
`````
This will look like

| This | is | a table |    |
|-----|---|--------|---|
|       |     |            |%  |

---

Do we have option to bypass this?

**Comment by ickc** [comment, marked as answer]
In (at least pandoc's) markdown, `%` has to be escaped (and I personally got encountered this mistakes from time to time.) i.e. write them as `\%`.

I bet if you use `markdown: False` then this won't be a problem.

Then the next question is if there should be an option to "intelligently" fix this common markdown mistakes in pandoc. I don't think it should, because as long as `markdown: True` then the cells should conform to (pandoc's) markdown spec so I don't think exceptional case should be made in this case.
